executor/aggregate_test.go

```diff
@@ -349,9 +349,101 @@ func (s *testSuite) TestAggPushDown(c *C) {
 	tk.MustQuery("select a, count(b) from (select * from t union all select * from tt) k group by a").Check(testkit.Rows("1 2", "2 1"))
 }
 
+func (s *testSuite) TestOnlyFullGroupBy(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("set sql_mode = 'ONLY_FULL_GROUP_BY'")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int not null primary key, b int not null, c int default null, d int not null, unique key I_b_c (b,c), unique key I_b_d (b,d))")
+	tk.MustExec("create table x(a int not null primary key, b int not null, c int default null, d int not null, unique key I_b_c (b,c), unique key I_b_d (b,d))")
+
+	// test AggregateFunc
+	tk.MustExec("select max(a) from t group by d")
+	// test not OnlyFullGroupBy
+	_, err := tk.Exec("select * from t group by d")
+	c.Assert(err, NotNil)
```

Comment:
We better check error equals `ErrFieldNotInGroupBy`

Reply:
ok